content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -663,6 +664,7 @@ Each feature gate is designed for enabling/disabling a specific feature:
   the volume mounter.
 - `MountPropagation`: Enable sharing volume mounted by one container to other containers or pods.
   For more details, please see [mount propagation](/docs/concepts/storage/volumes/#mount-propagation).
+- `NamespaceDefaultLabelName`: Enable the behavior of APIServer creating a default label `kubernetes.io/metadata.name` containing the namespace name.
```

Comment:
```suggestion
- `NamespaceDefaultLabelName`: Enable the APIServer to create a default label `kubernetes.io/metadata.name` containing the namespace name.
```

Reply:
We call it “the API server” or “kube-apiserver”.

We should mention _what_ gets labelled (ie, namespaces only, not every object in the namespace).

Reply:
+1

@tengqm if you change your suggestion here, I can commit directly :)

Reply:
Added here @sftim